src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -118,29 +118,18 @@ internal sealed class SourcePropertyAccessorSymbol : SourceMemberMethodSymbol
             NamedTypeSymbol containingType,
             SynthesizedRecordPropertySymbol property,
             DeclarationModifiers propertyModifiers,
-            string propertyName,
+            string name,
             Location location,
             CSharpSyntaxNode syntax,
             DiagnosticBag diagnostics)
         {
-            string name;
-            ImmutableArray<MethodSymbol> explicitInterfaceImplementations;
-            GetNameAndExplicitInterfaceImplementations(
```

Comment:
>GetNameAndExplicitInterfaceImplementations [](start = 12, length = 42)

I do not understand why this call is removed. It still does useful work of figuring out the name, that shouldn't be a responsibility of the caller. Please revert.

Reply:
Let's not duplicate the logic of figuring out initial name in every call site.

---
In reply to: [446525457](https://github.com/dotnet/roslyn/pull/45336#discussion_r446525457) [](ancestors = 446525457)